paper.md

```diff
@@ -0,0 +1,74 @@
+
+# Statement of need
+
+CLOVER was developed to provide a robust techno-economic and environmental analysis tool to support academics, practitioners and policymakers in evaluating sustainable energy interventions in developing countries. By providing an open-source alternative to proprietary minigrid design tools such as HOMER [@HOMER], CLOVER offers a more comprehensive analysis tool than common spreadsheet-based alternatives. CLOVER was designed to provide a fast, flexible and user-friendly software to assess energy systems at both short-term (hourly) and multi-year timescales under a wide variety of user-defined scenarios, constraints and optimisation conditions.
+
+CLOVER can provide an insight into the technical performance, costs, and environmental impact of a system, and allow the user to evaluate many different options to decide on the best way to provide sustainable, affordable and reliable electricity to the community. Its simulation functionality offers the user an insight into the hourly, daily, seasonal and multi-year technical performance of a system in response to changing electricity demands and varying renewable resources. Its optimisation process replicates the design process common in sustainable development settings by identifying optimum system design over short-term time horizons, but allowing the system to increase in capacity over its lifetime as the demand of the community grows in response to its economic development [@Sandwell:2017a].
+
+# Main features
+
+## Electricity supply
+CLOVER can simulate a variety of electricity supply technologies:
+- Solar generation is sourced from the Renewables.ninja API [@Renewables_Ninja] with the synthesis of the data described in @Pfenninger:2016;
+- Battery storage, charged from solar, with user-defined characteristics for battery performance and lifetime to emulate many kinds of technologies;
+- Diesel generation can be included as a source of power when no other source is available;
+- Grid power can be treated as a primary or backup source of electricity with a user-defined availability profile.
+These electricity sources can be included as the single source of electricity or in any combination as part of a hybrid system.
+
+## Electricity demand
+Users can generate electricity load profiles from the bottom up. Load profiles are based on the probability that individual user-specified appliances are in use, which can vary throughout the day and year, using processes described in @Sandwell:2016 and applied in @Chambon:2020. Any kind of electrical appliance and usage pattern can be defined by the user and ownership of appliances can be set to increase over time. Demands can be segregated into domestic, commercial, public, or other demand categories and included independently within the system. Alternatively users can provide their own hourly load profile, for example taken from monitored data.
```

Comment:
Suggestions:
"From the bottom up" - vauge - maybe "Users can generate electricity load profiles ~~from the bottom up~~ **based on ownership, usage and device-based information**"
Put backticks around "domestic," "commercial" and "public:"
Demands can be segregated ~~into `domestic`, `commercial`, `public`,~~ **into `domestic`, `commercial`, `public`,**

Reply:
Included as suggested